core.md

```diff
@@ -57,23 +57,13 @@ You may [apply to join some teams](https://links.neoforged.net/apply), but some
 
-They “steer” the project to keep it aligned with the long-term vision, hence its secondary name of Vision Team.
+In other words, the members of the Steering Council does not give out orders or demands. They cannot tell people what to do. Instead, if something comes up and a majority of the Steering Council members believes it is significant, they can say that the thing needs a decision from the whole team and put up a vote for that team. This helps teams be aware of what others members on their team are doing and helps reduce people being blindsided by big changes. It also addresses the problem of a single member making sweeping changes under other member's noses.
 
-The Council has at all times an odd number of active members, such that stalemates are not possible.
+The Steering Council has at all times an odd number of active members, such that stalemates are not possible when deciding if something warrants a vote or not. The members of the Steering Council are decided by an annual internal vote to ensure fairness and cycling of council members over time. Details on the Steering Council voting process can be found at [/steering-council-voting.md](/steering-council-voting.md).
 
-A member may choose to become inactive and appoint another in their place until they return.
-
-At least one member of the Council is promoted from other teams via an annual internal vote, to ensure that the Council always has fresh insight from a new team member with a different view on situations, and thus does not stagnate over time.
-
-Once a year, an internal voting process inspired by [the process implemented in the Python ecosystem](https://peps.python.org/pep-0013/) will determine the members that will be promoted to the Steering Council via the [Helios voting system](https://heliosvoting.org/). All three members of the Council will be elected, at least one of which must be a member that has not been on the Council in the previous term.
-
-The Council also serves double duty as a mediator for disagreements between other teams. These shall be resolved by discussion, rather than by unanimous applications of the rules, so this is not in the purview of this document.
-
-However, the Council does not have all-encompassing powers. They are the avenue of last resort, and the vast majority of decisions should be made by the team in charge, not by the Steering Council.
-
-The Council has these powers to resolve the 1% of exceptional situations, not the 99%.
+A member may choose to become inactive and appoint another Project team member in their place until they return. If a Steering Council member cannot be reached and there is an important time-sensitive Steering Council vote, an emergency Steering Council election will be held in accordance with [/steering-council-voting.md](/steering-council-voting.md). The newly elected member(s) will now act as the Steering Council member in question until their return or their term expires.
```

Comment:
Is this new?

Reply:
The old wording was `A member may choose to become inactive and appoint another in their place until they return.` which was vague and didn't account for if a member disappears without appointing a new member. There was talk in internal private which resulted in the general idea of this new paragraph to be more encompassing and handles edge cases better that the previous form did not cover.

Reply:
Gonna close this as you were just asking if it was new or not and I answered